ci-operator/step-registry/ipi/conf/vsphere/dns/ipi-conf-vsphere-dns-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: ipi-conf-vsphere-dns
+  from: openstack-installer
+  commands: ipi-conf-vsphere-dns-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 100Mi
+  documentation: |-
+    Uses VIPs in ${SHARED_DIR}/vips.txt to create route53 dns records. Also saves commands to be run during teardown.
```

Comment:
nit: can we define the inputs in a little detail here.. should be useful for people when they see it here like https://steps.svc.ci.openshift.org/registry/ipi-aws-pre